terminal/runereader_posix.go

```diff
@@ -1,3 +1,4 @@
+//go:build !windows
```

Comment:
Even though this is a newer syntax of the build directive, I do not think it's needed as it doesn't offer any improvements over `// +build !windows` and is only supported since Go 1.17.